platform/api/aws/ec2.go

```diff
@@ -240,27 +252,243 @@ func (a *API) GetConsoleOutput(instanceID string) (string, error) {
 // If the security group does not exist, it's created.
 func (a *API) getSecurityGroupID(name string) (string, error) {
 	sgIds, err := a.ec2.DescribeSecurityGroups(&ec2.DescribeSecurityGroupsInput{
-		GroupNames: []*string{&name},
+		Filters: []*ec2.Filter{
```

Comment:
Using the `group-name` Filter over the `GroupNames` parameter distances the check from default VPC's.

As a result this also allows us to drop the `isSecurityGroupNotExist` function.

Reply:
This would be a perfect comment to add to the code.